geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/index/RasterIndexEntry.scala

```diff
@@ -135,7 +136,8 @@ case class RasterIndexEntryDecoder() {
   def decode(entry: KeyValuePair) = {
     val renderedImage: RenderedImage = rasterImageDeserialize(entry._2.get)
     val metadata: DecodedIndex = RasterIndexEntry.decodeIndexCQMetadata(entry._1)
-    val res = 0.0 // TODO: WCS get the resolution from the row
-    Raster(renderedImage, metadata, 0.0)
+    val res = raster.lexiDecodeStringToDouble(new String(entry._1.getRowData.toArray).split("~")
```

Comment:
While out of scope of this ticket, I think some bits of the RasterIndexSchema could be written to do this. Please add a to-do here to that effect.